src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md

```diff
@@ -0,0 +1,22 @@
+---
+# Generated by scripts/aggregate-changelogs. WARNING: Manual edits to this files will be overwritten.
+aliases:
+- /changes/tenant-cluster-releases-vsphere/releases/vsphere-vsphere-28.0.0/
+changes_categories:
+- Workload cluster releases for VSPHERE
+changes_entry:
+  repository: giantswarm/releases
+  url: https://github.com/giantswarm/releases/tree/master/vsphere/v28.0.0
+  version: sphere-28.0.0
```

Comment:
@pipo02mix It trims off `v` which isn't desired here 😆

Reply:
hmm, that is a good catch. I will take a look now